DEVELOPING.md

```diff
@@ -4,7 +4,7 @@ Our development setup assumes a LINUX/BSD environemnt.
 
 ## Project Setup
 
-1. Install Node v14.
+1. Install Node v18.
 2. Install Yarn package manager `https://yarnpkg.com/en/docs/install` v1.10 or higher.
```

Comment:
any chance we will require a higher yarn version? is 1.10 compatible with node 14?

Reply:
it didnt seem necessary to change the yarn version, I would assume its compatible with node 14 since thats what we have been using.  unless you meant 18, I would assume it wouldnt since the min require eng is 14

Reply:
yeah, I meant whether yarn 1.10 is compatible with node 18, mb!